### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

File: src/cgame/cg_predict.cpp [hunk 1]
```diff
@@ -174,12 +174,12 @@ static void CG_ClipMoveToEntities( const vec3_t start, const vec3_t mins,
 		{
 		case traceType_t::TT_CAPSULE:
 		case traceType_t::TT_AABB:
-			CM_TransformedBoxTrace( &trace, start, end, mins, maxs, cmodel, mask, skipmask, origin, angles, collisionType );
+			CM_TransformedBoxTrace( &trace, start, end, mins, maxs, cmodel, MASK_ALL, 0, origin, angles, collisionType );
```
Comment (DolceTriade) [inline src/cgame/cg_predict.cpp:177]: `mask` + `skipmask` are now unused? We should remove the parameter or respect it.

**Review by DolceTriade — APPROVED** [review]

File: src/cgame/cg_predict.cpp (hunk 1 above)
Comment (slipher) [reply src/cgame/cg_predict.cpp:177]: CM_TransformedBoxTrace can be used to trace against either the world or a specific entity's bounding box. When tracing against the world the masks are useful, but a specific entity only has one content type so the masks are pointless.

Comment (DolceTriade) [reply src/cgame/cg_predict.cpp:177]: I think we should document this...

Comment (slipher) [reply src/cgame/cg_predict.cpp:177]: I realized the two different cases, tracing against a bbox or tracing against a map model, are actually combined in the same call sites. So I should only switch it to MASK_ALL for the bbox case. Anyway I will drop the commit from this PR for now and revisit later.

**Pull request merged**